### ISSUE TITLE
v2.0.0 (pre-release) Error on connection

### ISSUE THREAD

**Issue by XeiZ** [tsaibermelon/gg-struggle#17]
Hey,

whenever i launch Strive with the pre-release running it immediately closes the gg-struggle window, run through a elevated powershell i get this result:

![](https://i.imgur.com/aSoGwu1.png)

Went ahead and made sure every part that was mentioned was installed correctly but could very well be just a "me doing it wrong" thing, just making sure its not something that actually a problem and sneaks into the installer release.
Tried reinstalling everything, making sure it has read/write access to everything, is unblocked, no old certs from the previous versions remain.

**Comment by levibarrocas** [comment]
I am having the exact same issue right now

**Comment by cybermelons** [comment]
Thanks for bringing this up. Looks like its rejecting self signed certs for some reason

**Comment by cybermelons** [comment]
Yah this a bug

**Comment by priprii** [comment]
You can fix this by changing start.ps1 to:

```
$ScriptBlock = {
	$Env:NODE_TLS_REJECT_UNAUTHORIZED='0'
	node .
}
Invoke-Command -ScriptBlock $ScriptBlock
```

But then there's a part which throws another error:

```
[2021-07-26T20:50:46.964] [INFO] default - [DB] Dumping request to \AppData\Local\Temp/gg-struggle/dumps/b627588615559e35366585b10f1b5e2e203a4a7f.gameReq.dump

events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: connect ENOBUFS 127.0.0.1:443 - Local (undefined:undefined)
    at internalConnect (net.js:921:16)
    at defaultTriggerAsyncIdScope (internal/async_hooks.js:430:12)
    at net.js:1009:9
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (_http_client.js:469:9)
    at TLSSocket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -4060,
  code: 'ENOBUFS',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 443
}
```

Prior to that, my log is spammed with this:

```
[2021-07-26T20:46:07.656] [INFO] default - [PROXY] /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
[2021-07-26T20:46:07.657] [INFO] default - [DB] Dumping request to \AppData\Local\Temp/gg-struggle/dumps/b627588615559e35366585b10f1b5e2e203a4a7f.gameReq.dump
[2021-07-26T20:46:07.657] [INFO] default - [CACHE] Miss: /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
[2021-07-26T20:46:07.674] [INFO] default - [PROXY] /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
[2021-07-26T20:46:07.674] [INFO] default - [DB] Dumping request to \AppData\Local\Temp/gg-struggle/dumps/b627588615559e35366585b10f1b5e2e203a4a7f.gameReq.dump
[2021-07-26T20:46:07.674] [INFO] default - [CACHE] Miss: /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
[2021-07-26T20:46:07.680] [INFO] default - [PROXY] /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
[2021-07-26T20:46:07.680] [INFO] default - [DB] Dumping request to \AppData\Local\Temp/gg-struggle/dumps/b627588615559e35366585b10f1b5e2e203a4a7f.gameReq.dump
[2021-07-26T20:46:07.681] [INFO] default - [CACHE] Miss: /api/sys/get_env POST b627588615559e35366585b10f1b5e2e203a4a7f
```

It also additionally spams this one around 30 seconds after:

```
[2021-07-26T21:02:08.830] [INFO] default - [PROXY] /api/sys/get_news POST 722ed30712c19b713ff3a993598e1eb9d95fce65
[2021-07-26T21:02:08.830] [INFO] default - [DB] Dumping request to \AppData\Local\Temp/gg-struggle/dumps/722ed30712c19b713ff3a993598e1eb9d95fce65.gameReq.dump
[2021-07-26T21:02:08.830] [INFO] default - [CACHE] Miss: /api/sys/get_news POST 722ed30712c19b713ff3a993598e1eb9d95fce65
```

**Comment by cybermelons** [comment]
Fixed in #20. was due to the wrong dns cool to `lookup()` and not `resolve4()`
